pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -330,17 +348,41 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 		return nil, errors.Wrap(err, "failed to get cloud provider CA certificate")
 	}
 
+	transport := http.Transport{}
 	if userCACert != "" {
 		certPool, err := x509.SystemCertPool()
 		if err == nil {
 			certPool.AppendCertsFromPEM([]byte(userCACert))
-			client := http.Client{
-				Transport: &http.Transport{
-					TLSClientConfig: &tls.Config{
-						RootCAs: certPool,
-					},
-				},
+			transport.TLSClientConfig = &tls.Config{RootCAs: certPool}
+		}
+		client := http.Client{Transport: &transport}
+		provider.HTTPClient = client
+	}
+
+	// We cannot rely on the inject-proxy annotation because the CVO, which is
+	// responsible to inject the proxy env vars, runs on Pod Network.
```

Comment:
You're sure cluster-version-operator doesn't run on host networking? It's the first operator there is.

Reply:
yeah, it hanged on containerCreating and the ip was not one of the hosts